tests/test_rsa_keys.py

```diff
@@ -154,6 +157,31 @@ def test_verify_rsa_signature(self):
           scheme, public_rsa, data))
 
 
+  def test_verify_rsa_pss_sha256(self):
+    global public_rsa
+    global private_rsa
```

Comment:
I suppose you just copy/pasted these, but do we actually need `global` statements here?